src/service-module/service-module.mutations.ts

```diff
@@ -40,8 +40,9 @@ export default function makeServiceMutations() {
       }
 
       if (isTemp) {
-        if (!item[tempIdField]) {
-          var tempId = assignTempId(state, item)
+        let tempId = item[tempIdField];
+        if (!tempId) {
```

Comment:
Please change this to `if(tempId == null) {` so it will work with falsey, but valid, ids.